src/utils/sentry.ts

```diff
@@ -61,17 +61,49 @@ function initSentry() {
   // maskeAllText: mask all text on the screen.
   // ignoreErrors: Sentry ignores errors containing strings or regex. Regex needs to be exact message, string is partial
   // replaysSessionSampleRate: replays that begin recording immediately and last the entirety of the user's session.
-
+  // transport is what allows consumed apps to tell chrome what dsn to send things too.
   Sentry.init({
-    dsn: API_KEY,
+    //default api_key -> cp-001-insights
+    dsn: 'https://8b6372cad9604745ae3606bc4adc0060@o271843.ingest.sentry.io/1484024',
     environment: `Prod${appDetails.beta}`,
     maxBreadcrumbs: 50,
     attachStacktrace: true,
-    integrations: [new BrowserTracing(), new Sentry.Replay({ maskAllText: false, maskAllInputs: true })],
+    integrations: [
+      Sentry.browserTracingIntegration(),
+      Sentry.replayIntegration({ maskAllText: false, maskAllInputs: true }),
+      SentryBrowser.moduleMetadataIntegration(),
+    ],
     tracesSampleRate: 0.2,
     debug: !!window.localStorage.getItem('chrome:sentry:debug'),
     replaysOnErrorSampleRate: 1.0,
     replaysSessionSampleRate: 0.3,
+    transport,
+    beforeSend: (event) => {
+      if (event?.exception?.values?.[0]?.stacktrace?.frames) {
```

Comment:
Are the frames always limited to the first entry? Or can a relevant frame be in any exception? Furthermore, is it possible that more than one event should be modified?

Reply:
I spoke with the sentry folks and they confirmed it should always be the first entry. We wouldnt need to modify any other event. This section is just looking at the last frame to find witch project to send it too.